docs/resource-sharing/os-backfill-containers.md

```diff
@@ -159,14 +161,9 @@ modified to volume mount CVMFS instead of using cvmfsexec, and using reduced pri
 
 ```
 docker run -it --rm --user osg      \
-        --cap-add DAC_OVERRIDE      \
-        --cap-add DAC_READ_SEARCH   \
-        --cap-add SETUID            \
-        --cap-add SETGID            \
-        --cap-add SYS_ADMIN         \
-        --cap-add SYS_CHROOT        \
-        --cap-add SYS_PTRACE        \
-        -v /cvmfs:/cvmfs:shared     \
+        --security-opt seccomp=unconfined \
+        --security-opt systempaths=unconfined \
+        --security-opt no-new-privileges \
```

Comment:
Hold on, I just realized that you're also expecting unprivileged user namespaces here.  So the "important" popup needs to not just be in the cvmfsexec section, and this also needs --net=host.

Reply:
Moved this up to the `Before Starting` list of requirements